tests/EdgeToEdge/Routes/ApplyForMembershipRouteTest.php

```diff
@@ -178,4 +180,50 @@ public function testGivenValidRequest_requestIsRedirected() {
 		} );
 	}
 
+	public function testWhenApplicationGetsPersisted_timestampIsStoredInCookie() {
+		$this->createEnvironment( [], function ( Client $client, FunFunFactory $factory ) {
+			$factory->setNullMessenger();
+			$factory->setCookieHandler( new FixedCookieHandler( [ 'memapp_timestamp' => self::FIXED_TIMESTAMP ] ) );
```

Comment:
Right. When using a spy the test cannot check the actual timestamp stored, but this is covered in the other test anyway.